robottelo/cli/import_.py

```diff
@@ -89,10 +146,26 @@ def config_file(cls, options=None):
 
         """
         cls.command_sub = 'config-file'
-        return cls.execute(
+        result = cls.execute(
             cls._construct_command(options),
             output_format=''
         )
+        if result.return_code == 0:
+            result.transition_data = [
+                cls.read_transition_csv(
+                    ssh.command(
+                        u'ls -v ${HOME}/.transition_data/products*'
+                    ).stdout[:-1]
+                )
+            ]
+            result.transition_data.append(
+                cls.read_transition_csv(
+                    ssh.command(
+                        u'ls -v ${HOME}/.transition_data/puppet_repositories*'
+                    ).stdout[:-1]
+                )
+            )
```

Comment:
Please note, that some subcommands such as this one create and maintain 2+ files.
I return a list of dictionaries for those but this brings some output inconsistency between various subcommands.
I can "encapsulate" the output of the others to the list (making a 1-item list) if you guys think it'll be better.
Thanks for suggestions